### ISSUE TITLE
Failing Projection WKT Test with New Proj Library

### ISSUE THREAD

**Issue by geographika** [mapserver/mapserver#5817]
The build kit used by the AppVeyor CI for Windows from https://www.gisinternals.com/ uses Proj6. This is causing the following Python test to fail:

```python

    def testOGCWKT(self):
        self.map.setWKTProjection('PROJCS["unnamed", PROJECTION["Albers_Conic_Equal_Area"], '
                                  'PARAMETER["standard_parallel_1", 65], PARAMETER["standard_parallel_2", 55], '
                                  'PARAMETER["latitude_of_center", 0], PARAMETER["longitude_of_center", -153], '
                                  'PARAMETER["false_easting", -4943910.68], PARAMETER["false_northing", 0]]')
        proj4 = self.map.getProjection()

        assert proj4.find('+proj=aea') != -1
        assert proj4.find('+ellps=WGS84') != -1
        assert (mapscript.projectionObj(proj4)).getUnits() != mapscript.MS_DD
```

See https://ci.appveyor.com/project/SethG/mapserver/builds/25045014/job/2tc7ks7h1pgk6l9x for log output. 

The error is when loading the string - `MapServerError: msOGCWKT2ProjectionObj(): OGR error. Ingestion of WKT string`

Is there a new format that should be used here?

**Comment by rouault** [comment]
> Is there a new format that should be used here?

the WKT string is invalid. It lacks a GEOGCS node. Previous versions of GDAL were too lax regarding this. You could instead use the following for example
```
'PROJCS["unnamed", '
'GEOGCS["WGS 84",'
'DATUM["WGS_1984",'
'SPHEROID["WGS 84",6378137,298.257223563]],'
'PRIMEM["Greenwich",0],'
'UNIT["degree",0.0174532925199433]],'
'PROJECTION["Albers_Conic_Equal_Area"], '
'PARAMETER["standard_parallel_1", 65], PARAMETER["standard_parallel_2", 55], '
'PARAMETER["latitude_of_center", 0], PARAMETER["longitude_of_center", -153], '
'PARAMETER["false_easting", -4943910.68], PARAMETER["false_northing", 0]]'
```

**Comment by geographika** [comment]
Thanks @rouault. 
I also needed to add a `UNIT["metre",1.0]` option at the end, and check for `+datum=WGS84` rather than `+ellps=WGS84` in the test to get it to accept the WKT and pass. 
I've no idea what projection it is (if any), but based the WKT syntax on examples such as https://spatialreference.org/ref/sr-org/wkt-to-proj4-test/ogcwkt/
The test was to check the setWKTProjection() API works (and was added 15 years ago).
